apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -720,6 +734,22 @@ export class WorkshopForm extends React.Component {
     return value;
   };
 
+  // Selects the given value in the topic dropdown
+  handleSelect = event => {
+    const value = Object.keys(event)[0];
+    const isChecked = event[Object.keys(event)[0]];
```

Comment:
Nit: could this be `const isChecked = event[value];` ?

Reply:
Goooood call. Updated